pkg/controller/manager/manager_controller.go

```diff
@@ -521,6 +523,13 @@ func (r *ReconcileManager) Reconcile(ctx context.Context, request reconcile.Requ
 		replicas = &mcmReplicas
 	}
 
+	amz, err := utils.GetAmazonCloudIntegration(ctx, r.client)
+	if err != nil {
+		log.Error(err, "Failed to  fetch GetAmazonCloudIntegration info")
+	} else if amz != nil {
+		additionalNameSpaces = append(additionalNameSpaces, render.DexNamespace)
```

Comment:
I think this should be "tigera-amazon-cloud-integration"

Reply:
Done